plugin/trino-ranger/src/main/java/io/trino/plugin/ranger/RangerConfig.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ranger;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+public class RangerConfig
+{
+    private String keytab;
```

Comment:
Are you sure you want to add kerberos support in the first iteration? In order to make eligible for merging there will be needed test coverage for that and that requires product tests.

Product tests will be needed anyway, but with kerberos it is a bit harder. See `io.trino.tests.product.launcher.env.common.Kerberos`  and REDME in https://github.com/trinodb/trino/tree/master/testing/trino-product-tests. Here is the way how we run it: https://github.com/trinodb/trino/blob/8653030cc8325830bf13f7f0560a7b43feeb3be5/.github/workflows/ci.yml#L1005

Reply:
ok. Thank you for the pointer on testing with Kerberos. It is indeed harder; I removed Kerberos related configurations for the first iteration.